hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/HAUtils.java

```diff
@@ -285,4 +288,24 @@ public static File getMetaDir(DBDefinition definition,
     }
     return metadataDir;
   }
+
+  /**
+   * Unwrap exception to check if it is some kind of access control problem.
+   * {@link AccessControlException}
+   */
+  public static boolean isAccessControlException(Exception ex) {
```

Comment:
The same util exists in OmUtils, can we dedup the code with one copy?

Reply:
It has additional TokenValidation, for that OM has special logic. So, added a new method.
When SCM also has similar logic for TokenInvalid we can have a single copy of this method.